packages/vscode-extension/src/handlers.ts

```diff
@@ -718,6 +719,14 @@ export async function addFeatureHandler(args?: any[]): Promise<Result<null, FxEr
           await commands.executeCommand("markdown.preview.toggleLock");
         });
       }
+    } else if (workspacePath && result.value.func == UserTaskFunctionName.ConnectExistingApi) {
+      const components: string[] = result.value.component;
+      for (const item of components) {
+        const apiFilePath = path.join(workspacePath, item, result.value.fileName);
```

Comment:
can we  return the paths of generated sample code in api connector plugin? The logic in VS Code just need to open the files we return. In this way, if we want to do any change in the future (e.g. generate the files to another place), we just need to update api connector plugin.

Reply:
sure